x11-dl/src/link.rs

```diff
@@ -22,9 +22,9 @@ macro_rules! x11_link {
     globals:
     $(pub static $var_name:ident : $var_type:ty,)*
   } => {
+    #[non_exhaustive]
+    #[derive(Copy, Clone)]
```

Comment:
While I have no doubt there's a good internal reason for this, I am worried this might cause beginners to shoot themselves in the foot. What do you think is a good option?

Reply:
Okay, I removed `Copy` from the interface,